readme.md

```diff
@@ -46,9 +46,9 @@ Text to display after the spinner.
 
 ##### prefixText
 
-Type: `string`
+Type: `string | () => string`
 
-Text to display before the spinner. No prefix text will be displayed if set to an empty string.
+Text or text generator to display before the spinner. No prefix text will be displayed if set to an empty string.
```

Comment:
Not clear enough what a "text generator" is.

The docs changes here should also be reflected in index.d.ts

Reply:
How about "Text or function to return a string to display before the spinner"?

Reply:
Text or a function that returns text

Reply:
I have updated the readme and `index.d.ts` :)